### DISCUSSION TITLE
不懂就问，我的世界微软账号可以迁移到其他微软账号吗

### DISCUSSION THREAD

**Discussion by threeint** [Meloong-Git/PCL#1937] [提问] [answered]
如题

**Comment by Pigeon0v0** [comment]
可以同一账号换绑邮箱，但是貌似不能迁移（

**Comment by WForst-Breeze** [comment, marked as answer]
对于 Minecraft，迁移至微软账户后使用的账户应该就是 Xbox 账户了，你可以参考 [Xbox 给出的相关教程](https://support.xbox.com/zh-CN/help/account-profile/manage-account/change-email-sign-in-for-microsoft-account)来尝试更改您的邮箱账户。
